runtime/pack/dist/opt/netrw/plugin/netrwPlugin.vim

```diff
@@ -1,222 +1,207 @@
 " Maintainer: Luca Saccarola <github.e41mv@aleeas.com>
 " Former Maintainer: Charles E Campbell
 " Upstream: <https://github.com/saccarosium/netrw.vim>
-" Copyright:    Copyright (C) 1999-2021 Charles E. Campbell {{{1
+" Copyright:    Copyright (C) 1999-2021 Charles E. Campbell {{{
 "               Permission is hereby granted to use and distribute this code,
 "               with or without modifications, provided that this copyright
 "               notice is copied with it. Like anything else that's free,
 "               netrw.vim, netrwPlugin.vim, and netrwSettings.vim are provided
 "               *as is* and comes with no warranty of any kind, either
 "               expressed or implied. By using this plugin, you agree that
 "               in no event will the copyright holder be liable for any damages
-"               resulting from the use of this software.
+"               resulting from the use of this software. }}}
 
-" Load Once: {{{1
 if &cp || exists("g:loaded_netrwPlugin")
-  finish
+    finish
 endif
-let g:loaded_netrwPlugin = "v174"
-let s:keepcpo = &cpo
-set cpo&vim
```

Comment:
Those `cpo` settings (and the corresponding section at the end of the file should be kept